<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatME</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>681</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QWidget" name="pageLogin">
     <widget class="QLabel" name="labelWybierzNazwe">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wybierz nazwę:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
      </property>
      <property name="text">
       <string>Zaloguj</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelWybierzNazwe_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wybierz serwer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditNazwaUzytkownika">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-right-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditSerwer">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-right-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageChat">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgba(0,0,0,0);
font-size:20px;
</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonWyloguj">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>390</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
      </property>
      <property name="text">
       <string>Wyloguj</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditWiadomosc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-right-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserRozmowa">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>481</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-right-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Lista rozmówców&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>390</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
      </property>
      <property name="text">
       <string>Wyslij</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetUzytkownicy">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>120</y>
        <width>171</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-right-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelZalogowanyJako">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Zalogowany jako:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNazwaUzytkownika">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
